build.sbt

```diff
@@ -15,67 +15,50 @@
  */
 
 import Dependencies._
-import SparkVersionAxis._
 import JacocoSetup._
 import sbt.Keys.name
 
-
-ThisBuild / organization := "za.co.absa"
+ThisBuild / organization := "za.co.absa.atum-service"
+sonatypeProfileName := "za.co.absa"
 
 ThisBuild / scalaVersion := Versions.scala212  // default version
 
 ThisBuild / versionScheme := Some("early-semver")
 
 Global / onChangedBuildSource := ReloadOnSourceChanges
 
+publish / skip := true
 
 lazy val printSparkScalaVersion = taskKey[Unit]("Print Spark and Scala versions for atum-service is being built for.")
-ThisBuild / printSparkScalaVersion := {
-  val log = streams.value.log
-  val sparkVer = sparkVersionForScala(scalaVersion.value)
-  log.info(s"Building with Spark $sparkVer, Scala ${scalaVersion.value}")
-}
 
 lazy val commonSettings = Seq(
   libraryDependencies ++= commonDependencies,
   scalacOptions ++= Seq("-unchecked", "-deprecation", "-feature", "-Xfatal-warnings"),
   javacOptions ++= Seq("-source", "1.8", "-target", "1.8", "-Xlint"),
-  Test / parallelExecution := false
+  Test / parallelExecution := false,
+  jacocoExcludes := jacocoProjectExcludes()
 )
 
-val mergeStrategy: Def.SettingsDefinition = assembly / assemblyMergeStrategy := {
-  case PathList("META-INF", _) => MergeStrategy.discard
-  case "application.conf"      => MergeStrategy.concat
-  case "reference.conf"        => MergeStrategy.concat
-  case _                       => MergeStrategy.first
-}
-
-lazy val root = (projectMatrix in file("."))
-  .aggregate(model, server, agent)
-  .settings(
-    name := "atum-service-root",
-    javacOptions ++= Seq("-source", "1.8", "-target", "1.8", "-Xlint"),
-    publish / skip := true,
-    mergeStrategy
-  )
```

Comment:
Why is root not needed? / What are situations when it is needed? Just out of curiosity

Reply:
Interested as well. We use root in several projects.

Reply:
The root is there to pass some common configuration, but we use a different setup for that. So I cannot see any use for it, thus deleted. 

If you have situations where we need it, I am happy to get it back.